mmdet/models/utils/activations.py

```diff
@@ -0,0 +1,23 @@
+import torch
+from torch import nn
+
+
+class MemoryEfficientSwishImplementation(torch.autograd.Function):
+
+    @staticmethod
+    def forward(ctx, i):
+        result = i * torch.sigmoid(i)
+        ctx.save_for_backward(i)
+        return result
+
+    @staticmethod
+    def backward(ctx, grad_output):
+        i = ctx.saved_variables[0]
+        sigmoid_i = torch.sigmoid(i)
+        return grad_output * (sigmoid_i * (1 + i * (1 - sigmoid_i)))
+
+
+class Swish(nn.Module):
```

Comment:
We have Swish in mmcv/mmcv/cnn/bricks/swish.py. Need to distinguish these two.

Reply:
His is a memory-efficient version, which may be a bit different.

Reply:
Yes, I mean the class name should not be the same as the one in MMCV.